src/Kunstmaan/AdminBundle/Resources/translations/messages.en.yml

```diff
@@ -126,3 +134,8 @@ kuma_admin:
 kuma_js:
     auto_collapse:
         more_button_label: More
+
+toolbar:
+    exception:
+        title: Exceptions
+        message: "<strong>%X%</strong> exceptions within <strong>%Y%</strong> time"
```

Comment:
What I meant with ```X exceptions within X time``` was more in the sense of. An email will be sent out if you reach 20 exception in 5 minutes. That's what I meant with the configurable threshold. The usages of this translations also needs to change to ```X exception triggerd by X events```. Then the usage for this is correct.